cscs-checks/tools/io/nco.py

```diff
@@ -83,52 +88,58 @@ def setup(self, partition, environ, **job_opts):
         super().setup(partition, environ, **job_opts)
 
 
+@rfm.simple_test
 class InfoNCCheck(NCOBaseCheck):
     def __init__(self, **kwargs):
-        super().__init__('info_nc', **kwargs)
+        super().__init__('info_nc')
         self.descr = ('verifies reading info of a standard netCDF file')
         self.executable = 'ncks'
         self.executable_opts = ['-M', 'sresa1b_ncar_ccsm3-example.nc']
         self.sanity_patterns = sn.all([
             sn.assert_not_found(r'(?i)unsupported|error', self.stderr),
-            sn.assert_found(r'^Global attribute \d+: model_name_english',
+            sn.assert_found(r'model_name_english.*NCAR CCSM',
                             self.stdout)
         ])
 
 
+@rfm.simple_test
 class InfoNC4Check(NCOBaseCheck):
-    def __init__(self, **kwargs):
-        super().__init__('info_nc4', **kwargs)
+    def __init__(self):
+        super().__init__('info_nc4')
         self.descr = ('verifies reading info of a netCDF-4 file')
         self.executable = 'ncks'
         self.executable_opts = [
             '-M', 'test_echam_spectral-deflated_wind10_wl_ws.nc4'
         ]
         self.sanity_patterns = sn.all([
             sn.assert_not_found(r'(?i)unsupported|error', self.stderr),
-            sn.assert_found(r'^Global attribute \d+: CDO, size = 63 NC_CHAR',
+            sn.assert_found(r':physics.*"Modified ECMWF physics"|'
+                            r'^Global attribute \d+: CDO, size = 63 NC_CHAR',
                             self.stdout)
```

Comment:
Instead of having and or for two completely different expressions, could you maybe identify a common pattern between the previous output and the new output of `ncks -M test_echam_spectral-deflated_wind10_wl_ws.nc4`? Similar to what we found above in the `InfoNCCheck`, where we check that two strings appear in the same line? Maybe something with `NC_CHAR` that must be in the same line as something else?